platform/cc/RuntimeEffect.cc

```diff
@@ -0,0 +1,43 @@
+#include <jni.h>
+
+#include "SkRuntimeEffect.h"
+#include "interop.hh"
+
+JNIEXPORT jlong JNICALL
+Java_org_jetbrains_skija_RuntimeEffect__1nMakeShader(JNIEnv* env,
+                                                     jclass jclass,
+                                                     jlong ptr,
+                                                     jlong uniformPtr,
+                                                     jlongArray childrenPtrs,
+                                                     jlong childCount,
+                                                     jfloatArray localMatrixArr,
+                                                     jboolean isOpaque) {
+    SkRuntimeEffect* runtimeEffect = jlongToPtr<SkRuntimeEffect*>(ptr);
+    SkData* uniform = jlongToPtr<SkData*>(uniformPtr);
+    sk_sp<SkData> uniformData = SkData::MakeFromMalloc(uniform, uniform->size());
+
+    // Matrix
+    jfloat* m = env->GetFloatArrayElements(localMatrixArr, 0);
```

Comment:
Consider using skija::skMatrix, it does what you do here + handles null pointer in the place of an array